### DISCUSSION TITLE
SEB Providing this error message

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#965] [Q&A]
SEB showing this error message on a WIndows 11 device, unsure why

The error message that appears is SEB Detected a prohibited process which it was unable to exit: SVCHOST [OriginalName:]

![CAEC error](https://github.com/user-attachments/assets/c3e62139-bc00-4188-bf55-4ddc8aa5986e)

**Comment by dbuechel** [comment]
Because the configuration you're using has that process defined as a prohibited process, hence SEB detects and tries to terminate it.

**Reply by fkm97** [reply]
@dbuechel As far as I can tell, this process isn't listed as prohibited on the config file

**Reply by dbuechel** [reply]
Then it might be that a (system) service is trying to show a user interface (e.g. a dialog), which in turn triggers SEB's application monitoring. You may find more information in the client application log of SEB, to be found under `%LocalAppData%\SafeExamBrowser\Logs`.
